src/main/java/seedu/address/ui/PoliciesWindow.java

```diff
@@ -129,21 +134,37 @@ public void setPoliciesToDisplay(String name, String policiesToDisplay) {
             final String possibleUrl = policyNumAndUrl[1];
 
             if (isUrl(possibleUrl)) {
-                Button rowButton = new Button(COPY_URL_TEXT);
-                rowButton.setOnAction(e -> copyUrl(possibleUrl));
-                row.setAlignment(Pos.CENTER);
-                row.getChildren().addAll(new Label(split[i]), rowButton);
+                setupHBoxRowWithButton(row, split[i], possibleUrl);
             } else {
-                row.getChildren().addAll(new Label(split[i]));
+                setupHBoxRowNoButton(row, split[i]);
             }
             outerBox.getChildren().add(row);
         }
-        outerBox.setSpacing(10);
-        outerBox.setPadding(new Insets(25, 50, 25, 50));
+        formatVBox();
     }
 
-    private boolean hasPoliciesToDisplay(String[] split) {
-        return !split[0].equals(NO_POLICY_FEEDBACK);
+    private void clearWindow() {
+        outerBox.getChildren().clear();
+    }
+
+    private void setWindowTitle(String windowTitle) {
+        getRoot().setTitle(windowTitle);
+    }
+
+    private void setupHBoxRowWithButton(HBox row, String rowText, String url) {
+        Button rowButton = new Button(COPY_URL_TEXT);
+        rowButton.setOnAction(e -> copyUrl(url));
+        row.setAlignment(Pos.CENTER);
+        row.getChildren().addAll(new Label(rowText), rowButton);
+    }
+
+    private void setupHBoxRowNoButton(HBox row, String rowText) {
+        row.getChildren().add(new Label(rowText));
+    }
+
+    private void formatVBox() {
```

Comment:
Do you think that naming this method `formatOuterBox()` is more specific? since there are other VBox instances in other classes or if we extend the UI

Reply:
Good point. I've changed it!